client/package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "autoprefixer": "^10.4.12",
```

Comment:
curious - why do we need `autoprefixer`?

Reply:
good catch :+1: removed